awx_collection/tests/integration/targets/tower_token/tasks/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Generate names
+  set_fact:
+    token_description: "AWX-Collection-tests-tower_token-description-{{ lookup('password', '/dev/null chars=ascii_letters length=16') }}"
+
+- block:
+    - name: Create a Token
+      tower_token:
+        description: '{{ token_description }}'
+        scope: "write"
+        state: present
+      register: new_token
+
+    - name: Validate our token works by token
+      tower_job_list:
+        tower_oauthtoken: "{{ tower_token.token }}"
```

Comment:
I think this should be what was registered, and the documented return variable, `new_token.tower_token`

Reply:
We do document the return token as a dict (but not whats inside of the dict):
```
tower_token:
  type: dict
  description: A Tower token object which can be used for auth or token deletion
  returned: on successful create
```

Reply:
these tests still need to use `new_token` though right?

Reply:
oh nvm, looks like `tower_token` works, i didnt realize modules could just register variables but thinking about it now it makes sense